Ratings_MVP/Ratings_MVP/SearachFilmScreen/Presenter/SearchFilmPresenter.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SearchFilmPresenter.swift
+//  Ratings_MVP
+//
+//  Created by Валентин Коскин on 10.01.2023.
+//
+
+import Foundation
+
+/// Presenter for search films screen
+final class SearchFilmPresenter: SearchFilmPresenterProtocol {
+    weak var view: SearchFilmViewProtocol?
+    var networkService: NetworkServiceProtocol
+    var movies: [Movie]?
+    var router: RouterProtocol?
+
+    init(view: SearchFilmViewProtocol, networkService: NetworkServiceProtocol, router: RouterProtocol) {
+        self.view = view
+        self.networkService = networkService
+        self.router = router
+    }
+
+    // MARK: - Public Methods
+
+    func tapOnTheMovie(movie: Movie) {
+        router?.showMovieDetail(movie: movie)
+    }
+
+    func loadListOfMovies(filmUrl: String) {
+        networkService.loadFilmInformation(filmUrl: filmUrl) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(movies):
+                self.movies = movies[NetworkService.Constants.resultsText].arrayValue.map { Movie(json: $0) }
+                self.view?.success()
+            case let .failure(error):
+                self.view?.failure(error: error)
+            }
+        }
+    }
+
+    func loadMoviesListFromType(tag: Int) {
+        switch tag {
+        case 0: loadMoviesFromList(listFilmUrl: NetworkService.Constants.popularUrlText)
+        case 1: loadMoviesFromList(listFilmUrl: NetworkService.Constants.topRatingUrlText)
+        case 2: loadMoviesFromList(listFilmUrl: NetworkService.Constants.upcomingUrlText)
+        default: break
+        }
+    }
+
+    // MARK: - Private Methods
```

Comment:
```suggestion
    // MARK: - Private Methods
```
убери эту марку, у тебя ж там интерна